spec/requests/green_lanes/eligibilities_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'spec_helper'
+
+RSpec.describe GreenLanes::EligibilitiesController, type: :request do
+  subject { make_request && response }
+
+  before do
+    allow(TradeTariffFrontend).to receive(:green_lanes_enabled?).and_return true
+  end
+
+  let(:valid_params) do
```

Comment:
maybe we should move this inside the appropriate context

Reply:
yeah i can have a look at this i think this is here how i first had the tests set up